### ISSUE TITLE
Handle multiple files

### ISSUE THREAD

**Issue by ScreamingDev** [walle/gimli#40]
### Situation
- Gimli handles all Markdown in the folder
- Gimli handles one Markdown in the folder
### Problem
- Gimli does not handle 2,3,4,... files only
### Solution
- Let `-f` take more than one file (`gimli -f two.md three.md one.md`)
- _or_ let be there multiple `-f` (`gimli -f two.md -f three.md -f one.md`)

**Comment by floere** [comment]
This would be great to have. @walle Have you looked into it by any chance?

**Comment by walle** [comment]
Yes, I have, the thing is that I haven't had time (motivation really) to implement the feature.

The design of gimli is kind of broken from the start, I wrote it to be able to quickly generate nice pdfs for some reports that I was writing, and the use case then was to generate all files, or just one.

A better approach had been to use the common unix way eg. `cp SOURCE... DIRECTORY` like mdpdf (https://github.com/walle/mdpdf#usage) does. To take one or many files (or stdin) and write it to a file (or stdout). This gives a lot more flexibility and makes it possible to chain the application using pipes, which would be nice. Please note that the mdpdf application is a quick project I wrote to get back my C legs and doesn't have all the features gimli does.

I have started writing a specification that should take into account all the use cases and properly document the features available, I have some work to do left though. But my idea is to ask the community (~350 stars so hopefully I will get some response) to read the specification and  improve on it. When the specification is set implement it. The specification would give some structure to the development, eg. changes would be discussed in regards to the spec before implementing them. The more structured development (I think) would give me more motivation to work on the project.

So this change would enable this feature (and give some extra perks).

**Comment by floere** [comment]
Thanks for your explanation :)
